index.bs

```diff
@@ -2802,13 +3038,29 @@ The [=remote end steps=] with |command parameters| are:
   1. If the implementation is unable to create a new browsing context for any
      reason then return [=error=] with [=error code=] [=unsupported operation=].
 
+  1. Let |user context| be the [=default user context=] if |reference context| is null, and |reference context|' [=user context=] otherwise.
+
+  1. If |command parameters|[<code>userContext</code>] is present and is not null:
+
+    1. Let |user context id| be |command parameters|[<code>userContext</code>].
+
+    1. If the [=user context=] with the [=user context id=] equal to |user context id| does
+      not exist in the [=remote end=], return [=error=] with [=error code=] [=no such user context=].
+
+    1. Set |user context| to the [=user context=] with the [=user context id=] equal
+      to |user context id|.
+
+    1. If |reference context| is not null and |reference context|'s [=user context=] is not |user context|,
```

Comment:
I think we should have something more generic e.g. moving the first step down and saying something like:

```
1. If the implementation is unable to create a new [=top-level traversable=] with [=associated user context=] |user context| for any reason, then return [=error=] with [=error code=] [=unsupported operation=].
```

That makes the error for the specific case of "can't create a user context in the wrong window in Chrome" a generic "unsupported operation" error rather than "invalid user context".